resource_templates/iam_policy.tf

```diff
@@ -0,0 +1,29 @@
+// Create/ Delete/ Update Tanzu Mission Control iam policy entry
+resource "tanzu-mission-control_iam_policy" "create_iam_policy" {
+  scope {
+    cluster{
+      management_cluster_name = "<management-cluster>" // Required
+      provisioner_name        = "<prov-name>"          // Required
+      cluster_name            = "<cluster-name>"       // Required
+    }
+  }
+
+  role_bindings{
+    role = "<user-role>"
+    subjects {
+      name = "<user-identity>"
+      kind = "<identities>"
+    }
+  }
+  role_bindings{
+    role = "<user-role>"
+    subjects {
+      name = "<user-identity>"
+      kind = "<identities>"
+    }
+    subjects {
+      name = "<user-identity>"
+      kind = "<identities>"
+    }
+  }
+}
```

Comment:
Can we add this for documentation in a separate PR or at least as a separate commit?